docs/src/man/reshaping_new.md

```diff
@@ -0,0 +1,498 @@
+# Reshaping Data
+
+*DataFrames.jl* is a popular julia library for tabular data which makes easier to read and transform data.
+It provides the abstractions of data frames and series, similar to those in *R*, and *Pandas*.
+
+In *DataFrames.jl* data reshaping means the transformaton of the structure of the table or vector (for example 
+data frames or series) to make it suitable for further analysis. Let us assume we have a data frame with 
+multiindices on the rows and columns. In `DataFrames.jl` currently we have two functions: `stack`, and `unstack`
```

Comment:
Where did you get this part from? In DataFrames.jl we do not support such functionalities.